src/core/call.js

```diff
@@ -11,16 +11,13 @@
  * @param token
  * requiresAuthentication: boolean
  */
-export const call = (method, path, urlParams, body, token) => {
-  const obj = {
-    method,
-    path
-  }
-  if (urlParams) { obj.urlParams = urlParams }
-  if (body) { obj.body = body }
-  if (token) { obj.token = token }
-  return obj
-}
+export const call = (method, path, urlParams, body, token) => ({
+  method,
+  path,
+  ...urlParams,
+  ...body,
+  ...token
```

Comment:
Good catch. Changed to use this syntax and some tests to check that it works
```
export const call = (method, path, urlParams, body, token) => ({
  method,
  path,
  ...urlParams && { urlParams },
  ...body && { body },
  ...token && { token }
})
```
It is actually how this is recommended [here](http://stackoverflow.com/questions/11704267/in-javascript-how-to-conditionally-add-a-member-to-an-object#40560953). I missed that.